src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.Stream.cs

```diff
@@ -60,7 +60,7 @@ public static partial class JsonSerializer
         public static ValueTask<object> DeserializeAsync(
```

Comment:
This can return a nullable object as part of the task (i.e. should be `ValueTask<object?>`).

```C#
var stream = new MemoryStream(Encoding.UTF8.GetBytes("null"));
object nullObj = await JsonSerializer.DeserializeAsync(stream, typeof(string));
```